lib/moby_derp/pod.rb

```diff
@@ -38,9 +38,9 @@ def run
 				begin
 					MobyDerp::Container.new(pod: self, container_config: cfg).run
 				rescue MobyDerp::ContainerError => ex
-					raise MobyDerp::ContainerError,
-					      "error while running container #{cfg.name}: #{ex.message}",
-							ex.backtrace
+					@logger.error(logloc) {
```

Comment:
This change seems unrelated to the purpose of the other changes in this PR; it would be better to submit as a separate PR as I have some potential reservations, and I don't want to hold up these other changes debating it.